### DISCUSSION TITLE
Bug: Using template tag inside tbody

### DISCUSSION THREAD

**Discussion by nfplee** [vuejs/petite-vue#124] [General]
I think I've found a bug, I can't seem to use the template tag within a tbody tag. Here's the broken version using petite-vue:

https://jsfiddle.net/3enusvtr/1/

Here's the working version which uses the full framework:

https://jsfiddle.net/q6uxyswc/1/

Please note that my example has been simplified to illustrate the problem.

**Comment by halivert** [comment]
Yes, it's not a bug, I think the full Vue behavior can't be reproduced because petite-vue doesn't use a virtual DOM, and you can't use a `v-if` in the first child of a `<template>` element.

Anyway you can workaround and do something like this https://jsfiddle.net/halivert/zc0sgxk5/4/

**Reply by nfplee** [reply]
Thanks, your workaround is nicer than the original syntax.
